content/posts/0.26-release.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "0"
+date: 2024-05-13T11:00:55-04:00
+draft: true
+tags: ["announcement"]
+# author: "Me"
+# author: ["Me", "You"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+description: "Desc Text."
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    # image: "<image path/url>" # image path/url
+    # alt: "<alt text>" # alt text
+    # caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+Pyodide v0.26.0 is out with Python 3.12. We made a lot of minor improvements to
+the FFI and to the build system, and we made major improvements to stack
+switching support. Most of the changes we made for this release are forward
+looking, either nonfunctional work on maintenance, or functional work that is
+still not exposed to the end user.
+
+## Package build system
+
+We made made major progress in unvendoring our package build system. This is
+critical work because the current system of building 250+ packages on every
+commit is not sustainable. ...
+
+## JS API docs renderer Improvements
```

Comment:
I think people will be interested in sphinx-js. But I agree with you it would be nice if it had any explanation of what the goals are. "Improved document maintenance".